js/contacts.js

```diff
@@ -1,41 +1,45 @@
+document.addEventListener("DOMContentLoaded", function(event) {
 //Everything will be for working with contacts
 var baseUrl = OC.generateUrl('/apps/sciencemesh');
-$('#test_error').hide(); 
+
+$('#test').hide(); 
 $.ajax({
     url: baseUrl + '/contacts/users',
     type: 'GET',
     contentType: 'application/json',
 }).done(function (response) {
-    if(response === '' || response === false) {
-        var element = document.getElementById("test_error");
-        element.innerHTML= 'No Sciencemesh Connection';
-        $('#test_error').show(); 
-    } else {
+
+    var headerElement = document.getElementById("message");
     let token = JSON.parse(response);
+    if(response === '' || response === false || token['accepted_users'] === undefined) {
+        headerElement.innerHTML= 'No Reva Contact';
+        $('#test').show(); 
+    } else {
+        headerElement.innerHTML= 'Reva Contacts';
+
```

Comment:
These changes seem unrelated to "add federated share type to sciencemeshShareProviderFactory"?